odxtools/nameditemlist.py

```diff
@@ -49,6 +50,11 @@ def append(self, item: T):
                           f"Encountered name '{item_name}' which is not an "
                           f"identifier!")
 
+        # make sure that the name of the item in question is not a
+        # python keyword (this would lead to syntax errors)
+        if iskeyword(item_name):
+            item_name = f"{item_name}_"
```

Comment:
does this affect trying to access value with `[“name“] `?

Reply:
yes, but that already happens for names that are not valid python identifiers (like `4ever`, `2do`, etc.) The main purpose of this class is to make interactive exploration of the dataset as convenient as possible, so this is IMO okay. 

The bullet-proof way of accessing parameters is to go through the list directly, e.g.
```python
nil = NamedItemList(lambda x: x[0], [("foo", 1), ("bar", 2)])
foo = [ x for x in nil if x[0] == "foo"][0]
print(foo) # ("foo", 1)
```